owncloudApp/src/test/java/com/owncloud/android/presentation/viewmodels/security/PatternViewModelTest.kt

```diff
@@ -36,13 +37,15 @@ import org.junit.Test
 class PatternViewModelTest : ViewModelTest() {
     private lateinit var patternViewModel: PatternViewModel
     private lateinit var preferencesProvider: SharedPreferencesProvider
+    private lateinit var contextProvider: ContextProvider
 
     private val pattern = "1234"
 
     @Before
     fun setUp() {
         preferencesProvider = mockk(relaxUnitFun = true)
-        patternViewModel = PatternViewModel(preferencesProvider)
+        contextProvider = mockk(relaxUnitFun = true)
+        patternViewModel = PatternViewModel(preferencesProvider, contextProvider)
```

Comment:
This will remain as before, as `ContextProvider` won't be necessary anymore 👍

Reply:
Done